packages/driver/src/cy/commands/actions/click.ts

```diff
@@ -117,11 +117,35 @@ export default (Commands, Cypress, cy: $Cy, state, config) => {
         // figure out the options which actually change the behavior of clicks
         deltaOptions = $utils.filterOutOptions(options, defaultOptions)
 
-        options._log = Cypress.log({
-          message: deltaOptions,
-          $el,
-          timeout: options.timeout,
-        })
+        // create a new object with all user-provided options and exclude null values
+        const clickOptions = Object.keys(userOptions)
+        .reduce((obj, key) => {
+          if (options[key] !== null) {
+            obj[key] = options[key]
+          }
+
+          return obj
+        }, {})
+
+        // Add the position property to the clickOptions object if it's not null or undefined
+        if (options.position !== null && options.position !== undefined) {
+          clickOptions.position = options.position
+        }
+
+        // Only log the message if the clickOptions object has any properties
+        if (Object.keys(clickOptions).length > 0) {
```

Comment:
Should be able to remove/simplify this if-else statement to something like the following which handles adding details to the log message specifically:
```js
let message = []

if (userOptions) {
  const clickOptions = ....
  message.push(clickOptions)
}

options._log = Cypress.log(...)
```

Reply:
I have refactored the code to address the log feature comment & this comment is not longer relevant.